notebook/prepare.sh

```diff
@@ -4,17 +4,13 @@ set -x
 
 echo "Copy Dask configuration files from pre-load directory into home/.config"
 mkdir -p /home/jovyan/.config/dask
```

Comment:
We can drop this line right? It will be implied by the copy of the folder contents?